tests/ai_ml/armnn.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: This module install armnn and downloads
+#   test programs, models, labels and images.
+#   Then, it runs models with the images as inputs.
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub armnn_get_images {
+    assert_script_run('mkdir data && pushd data');
+    assert_script_run('wget https://images.theconversation.com/files/207813/original/file-20180226-140181-un3yf6.jpg -O shark.jpg');
+    assert_script_run('wget https://faqcats.com/wp-content/uploads/tabby-cats-vs-tiger-cats-300x205.jpg -O Cat.jpg');
+    assert_script_run('wget https://upload.wikimedia.org/wikipedia/commons/2/26/YellowLabradorLooking_new.jpg -O Dog.jpg');
+    assert_script_run('popd');
+}
+
+sub armnn_tf_lite_test_prepare {
```

Comment:
isn't it better to put all this inside a shell script and just verify that the files exist in  the end?

Reply:
> isn't it better to put all this inside a shell script and just verify that the files exist in the end?

You mean a script in `./data/` ?

Reply:
yep

Reply:
On it.